devices/pine64-pinephone/kernel/default.nix

```diff
@@ -22,6 +22,11 @@ mobile-nixos.kernel-builder {
       url = "https://github.com/mobile-nixos/linux/commit/372597b5449b7e21ad59dba0842091f4f1ed34b2.patch";
       sha256 = "1lca3fdmx2wglplp47z2d1030bgcidaf1fhbnfvkfwk3fj3grixc";
     })
+    # Drop modem-power from DT to allow eg25-manager to have full control.
```

Comment:
Can we do this without a patch somehow?

Reply:
I'll test `CONFIG_MODEM_POWER=n`.

Reply:
As a first attempt, just unsetting `CONFIG_MODEM_POWER` was not enough: eg25-manager reports the modem doesn't come up. https://paste.debian.net/1234008/

Reply:
I'll resolve this, since we're now doing what other distros are doing.